### DISCUSSION TITLE
GPG sign tags created from releases

### DISCUSSION THREAD

**Discussion by spenserblack** [community/community#11690] [Other Feature Feedback, Questions, & Ideas]
When a release creates a new tag, the tag doesn't have a GPG signature. I think that, just like how commits created through the frontend are signed, tags should be signed too. This verifies that the creator of the tag was authenticated by GitHub.

Edit: Apologies for the duplicate discussion. I didn't see #5508 in my initial search, but found it after creating this discussion.

**Comment by dotysan** [comment]
Have you tried `git config --global tag.gpgsign true`?

https://git-scm.com/docs/git-config#Documentation/git-config.txt-taggpgSign

**Reply by spenserblack** [reply]
Yes, I do that locally and then push up a tag. I mean signing a tag that is created *by GitHub*, by creating a release that would create a tag either through the web service or through the API.
